julia/diseaseProg.jl

```diff
@@ -426,3 +428,37 @@ function _einsum3(a, b) #'ijkl,j->i'
     end
     return sum(p, dims=1)
 end
+
+function trFunc_travelInfectionRate_ageAdjusted(
+	t::Int64, # Time within simulation
+
+	travelMaxTime::Int64 = 200,
+	travelBaseRate::Float64 = 5e-4, # How many people normally travel back to the country per day # TODO - get data
+	travelDecline_mean::Float64 = 15.,
+	travelDecline_slope::Float64 = 1.,
+
```

Comment:
```suggestion
```

Reply:
Blank spaces should be avoided and you can write `1.0` instead of `1.` which looks better:)

Reply:
Okay..I will change it